docs/REST API/rest-api.md

```diff
@@ -5326,6 +5326,8 @@ If `mdm_id`, `mdm_name`, `mdm_enrollment_status`, `os_settings`, or `os_settings
 
 ### Delete label
```

Comment:
### Dev note

Error message returned in the API when the user attempts to delete a label that's being used as a custom target for software. 

Here's what this error message will look like in the UI: https://www.figma.com/design/4pfUOYy7IyMIrjMH2fuCdU/%2319551-Automatically-install-software-and-scope-software-with-labels?node-id=5493-9746&t=piN5Gu8vzYRvESPT-1

Reply:
TODO: Remove this comment before merge

Reply:
@RachelElysia here's the error message we chatted about on our call today.

FYI @lucasmrod